File: PRAK604-2210817220011-MeldianaHandayani.py
k1=input(); k2=input()
kt1=len(k1); kt2=len(k2)
i=0; j=0
if kt1!=kt2:
    print("Panjang kalimat berbeda, pesan palsu")
else:
    for k in range (0,kt1):
        if k1[k]==k2[k]:
            if k1[k]=='':
                print(" ", end='')
            else:
                print("*", end='');i+=1
        else:
            print("#", end=''); j+=1
    print("\n* = ", i)
    print("# = ", j)
    if i>=j:
        print("Pesan Asli")
    else:
        print("Pesan Palsu")
    

File: PRAK604-2210817220011-MeldianaHandayani.c
#include <stdio.h>
#include <string.h>
int main(){
    int i,j;
    char k1[50], k2[50];
    printf(""); gets(k1);
    printf(""); gets(k2);
    int kata1=strlen(k1), kata2= strlen(k2);
    if (kata1!=kata2){
        printf("Panjang Kalimat berbeda, pesan palsu");}
    else{
        int i=0, j=0;
        for (int k=0; k<kata1;k++){
            if (k2[k]==' '&&k1[k]==' '){
                continue;}
            else if(k1[k]!=k2[k]){
                printf("#");
                i++;}
            else if(k1[k]==k2[k]){
                printf("*");
                j++;}}
        printf("\n*= %d",j);
        printf("\n#= %d",i);
        if(i<=j){
            printf("\nPesan Asli");}
        else{
            printf("\nPesan Palsu");}}
}